apps/tenant-management-webapp/src/app/pages/admin/services/form/form.tsx

```diff
@@ -0,0 +1,61 @@
+import React, { FunctionComponent, useState } from 'react';
+import { useSelector } from 'react-redux';
+import FormOverview from './formOverview';
+import { RootState } from '@store/index';
+import { Aside, Main, Page } from '@components/Html';
+import { Tab, Tabs } from '@components/Tabs';
+import SupportLinks from '@components/SupportLinks';
+
+const HelpLink = (): JSX.Element => {
+  const tenantName = useSelector((state: RootState) => state.tenant?.name);
+  const docBaseUrl = useSelector((state: RootState) => state.config.serviceUrls?.docServiceApiUrl);
+  return (
+    <>
+      <h3>Helpful links</h3>
```

Comment:
Since Rama working on  helpful link in all other screen, please make sure have consistency with her work, thanks!